qm-plugins/qm-object-cache/css/style.css

```diff
@@ -1,69 +1,69 @@
 #query-monitor-main .qm-hide-operation, #query-monitor-main .qm-hide-group {
-    display: none !important;
-  }
+				display: none !important;
+		}
```

Comment:
Looks like each one space was replaced with one tab, which now means this is indented 4 tabs in (should be 1) and two tabs (should be 0).

The other lines are affected too.